versions.tf

```diff
@@ -1,3 +0,0 @@
-terraform {
```

Comment:
I think we'll want to keep this block. It sets a minimum version of terraform that is required for the project, which is part of the recommended conventions for terraform modules. The filename, `versions.tf`, comes from the terraform "upgrade" commands, which are designed to help upgrade from one version of terraform to another. The upgrade commands will create a versions.tf file if one does not already exist.

Reply:
Renamed providers.tf to versions.tf.